service/library/setup.go

```diff
@@ -89,11 +89,11 @@ func Setup(c *Config) {
 			Name:    "MacOS Resolver",
 		}
 
-		var ResolvGeneric = resolv.Resolv{
-			Data:    "nameserver 127.0.0.1 # added by amazee.io pygmy",
+		var ResolvLinux = resolv.Resolv{
+			Data:    "# Generated by amazeeio pygmy\n[Resolve]\nDNS=127.0.0.1:6053\nDomains=docker.amazee.io",
```

Comment:
Could we newline at the end?

Reply:
Resolved in 9bdcaeeda54d013c06e8539358dce4fb0c219499